superset-frontend/src/explore/components/controls/SelectControl.jsx

```diff
@@ -149,6 +158,15 @@ export default class SelectControl extends React.PureComponent {
         return { value: c, label: c };
       });
     }
+
+    if (allowAll === true) {
```

Comment:
I'm okay with this, but just wondering if JS is like Java and we could get away with just "if(allowAll) {"